receiver/cloudfoundryreceiver/config_test.go

```diff
@@ -80,7 +80,7 @@ func TestLoadInvalidConfig(t *testing.T) {
 
 	factory := NewFactory()
 	factories.Receivers[typeStr] = factory
-	_, err = servicetest.LoadConfigAndValidate(filepath.Join("testdata", "config-invalid.yaml"), factories)
+	_, err = servicetest.LoadConfigAndValidate(filepath.Join("testdata", "config-invalid-datatype.yaml"), factories)
```

Comment:
is this a stale update?

Reply:
not sure -- what makes you think it's stale?

Reply:
Oh I see. I changed that filename and GoLand probably updated that string. Will take a look (definitely don't want to be updating the cloud foundry receiver 😄).  Good catch.

Reply:
Should be reverted now.